cfgov/unprocessed/css/molecules/notification.less

```diff
@@ -0,0 +1,101 @@
+/* ==========================================================================
+   Capital Framework
+   Notification Styling
+   ========================================================================== */
+
+
+/* topdoc
+  name: Notifications
+  family: cf-molecules
+  notes:
+    - "Markup and styling for notifications"
+  patterns:
+    - name: Default Usage
+      markup: |
+        <div class="m-notification m-notification__error">
+          <span class="cf-notification_icon
+                       cf-notification_icon__error
+                       cf-icon
+                       cf-icon-error-round"></span>
+          <div class="m-notification_message">
+              This is a default alert.
+          </div>
+          <div class="m-notification_explanation">
+              This is a default explanation with a <a href="#">link</a>.
+          </div>
+        </div>
+  tags:
+    - cf-molecules
+*/
+
+@m-notification-padding__px: 15px;
+
+.m-notification {
+    /* display: none; TODO: uncomment this line when prototype is merged into filters. x*/
+    position: relative;
+
+    padding: @m-notification-padding__px;
+    padding-left: 40px;
+    margin-bottom: @m-notification-padding__px + 4px;
```

Comment:
Hm, this actually should be there at all. A notification shouldn't be aware of its margin. Removed in https://github.com/cfpb/cfgov-refresh/commit/e1a7a023f616f09735e4038ab6a9ba8c50cfcbc6